index.html

```diff
@@ -15,22 +15,67 @@
             <img src="./img/close.png" id="mob-menu" alt="close mobile version" class="mob-click">
```

Comment:
Your mobile hamburger button is on the left side and it works like a charm 😎 Kindly make sure that your hamburger button image is transparent so that the design is not compromised by its white background 🚀

![image](https://user-images.githubusercontent.com/15807343/164559289-15efe698-b1e3-4814-907d-9ea609dc37ff.png)